openstack/designate/templates/producer-deployment.yaml

```diff
@@ -75,18 +75,16 @@ spec:
           livenessProbe:
             exec:
               command:
-              - bash
-              - -c
-{{- if .Values.global_setup }}
-              - "set -e; nc -zvw3 designate-global-percona-pxc 3306; nc -zvw3 designate-global-memcached 11211"
```

Comment:
For producer deployment - yes, because the check relies on rpc probe and RabbitMQ connection.

For API/Central it works the same way but relies on the liveness/readiness status of the corresponding percona-global pod and related service. Unfortunately, it looks like this readiness probe in percona-global doesn't fail when the node goes into the partitioned state, so all these TCP check always succeed except during a node startup.